src/main/kotlin/no/nav/faktureringskomponenten/service/avregning/AvregningBehandler.kt

```diff
@@ -38,7 +38,9 @@ class AvregningBehandler(private val avregningsfakturaGenerator: Avregningsfaktu
         if (avregningsperioder.isEmpty()) return emptyList()
         log.debug { "Avregningsperioder generert: $avregningsperioder" }
 
-        return avregningsperioder.map {
+        return avregningsperioder.filter {
+            (it.nyttBeløp > BigDecimal.ZERO && it.tidligereBeløp < BigDecimal.ZERO && it.bestilteFaktura.erAvregningsfaktura()) || it.nyttBeløp === BigDecimal.ZERO
```

Comment:
Skjønner ikke helt hva vi prøver å få til her. Trenger nok forklarende metodenavn eller kommentar.
Hvorfor er `bestilteFaktura.erAvregningsfaktura nødvendig`?

`it.nyttBeløp == BigDecimal.ZERO` er vel feil hvis `it.tidligereBeløp` er et minusbeløp samtidig